plugin.xml

```diff
@@ -140,23 +140,31 @@
 
   <!-- android -->
   <platform name="android">
-    <config-file target="AndroidManifest.xml" parent="/manifest/application">
+
+    <!-- Fixes the activity tag in the Manifest -->
+    <edit-config file="AndroidManifest.xml" target="/manifest/application/activity" mode="merge">
       <activity android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale"
           android:label="@string/app_name"
           android:name="com.salesforce.androidsdk.phonegap.ui.SalesforceDroidGapActivity"
-          android:theme="@android:style/Theme.Black.NoTitleBar">
-        <intent-filter>
-          <action android:name="android.intent.action.MAIN" />
-          <category android:name="android.intent.category.LAUNCHER" />
-        </intent-filter>
-      </activity>
-    </config-file>
+          android:theme="@android:style/Theme.Black.NoTitleBar"
+          android:launchMode="singleTop" />
+    </edit-config>
+
+    <!-- Fixes the application tag in the Manifest -->
+    <edit-config file="AndroidManifest.xml" target="/manifest/application" mode="merge">
+      <application android:hardwareAccelerated="true"
+          android:icon="@drawable/sf__icon"
+          android:label="@string/app_name"
+          android:manageSpaceActivity="com.salesforce.androidsdk.ui.ManageSpaceActivity"
+          android:name="com.salesforce.androidsdk.phonegap.app.HybridApp" />
+    </edit-config>
 
-    <config-file target="res/xml/config.xml" parent="/*">
+    <!-- Fixes the min and target SDK versions in the Manifest -->
+    <edit-config file="AndroidManifest.xml" target="/manifest/uses-sdk" mode="merge">
```

Comment:
`edit-config` for right `minSdkVersion` and `targetSdkVersion`. This seems more reliable than using `<preference>` tags.